server/apiv2/router.go

```diff
@@ -51,9 +51,9 @@ func NewV2Handler(_ context.Context, svr *server.Server) (http.Handler, apiutil.
 		c.Set(middlewares.ServerContextKey, svr)
 		c.Next()
 	})
-	router.Use(middlewares.Redirector())
 	root := router.Group(apiV2Prefix)
 	root.GET("ready", handlers.Ready)
+	root.Use(middlewares.Redirector())
```

Comment:
How about adding a comment such as `Register /ready endpoint before Redirector middleware to exclude it from redirection logic?